### ISSUE TITLE
RtContainersTestCase.java:49-52: Impediment: once #45 is...

### ISSUE THREAD

**Issue by 0pdd** [amihaiemil/docker-java-api#54]
The puzzle `47-dbe85f1e` from #47 has to be resolved:

https://github.com/amihaiemil/docker-java-api/blob/7d8c9980f004f50fdc22b2024f97b9253629bcf7/src/test/java/com/amihaiemil/docker/RtContainersTestCase.java#L49-L52

The puzzle was created by George Aristy on 15-Mar-18. 

Estimate: 30 minutes, role: DEV.

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@amihaiemil[/z](https://www.0crat.com/u/amihaiemil) please, pay attention to this issue

**Comment by amihaiemil** [comment]
@0crat in

**Comment by 0crat** [comment]
> @0crat in ([here](https://github.com/amihaiemil/docker-java-api/issues/54#issuecomment-373636835))

@amihaiemil Job [#54](https://github.com/amihaiemil/docker-java-api/issues/54) is now in scope, role is `DEV`

**Comment by amihaiemil** [comment]
@llorllale are you on vacation? How come 0crat doesn't assign the tasks? :D

**Comment by llorllale** [comment]
@amihaiemil no i'm not.

**Comment by amihaiemil** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/amihaiemil/docker-java-api/issues/54#issuecomment-373937844))

@amihaiemil This is what I know about this job, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#54](https://github.com/amihaiemil/docker-java-api/issues/54) is [in scope](http://www.zerocracy.com/policy.html#14) for 1days
  * The role is `DEV`
  * The job is not assigned to anyone
  * The [budget](http://www.zerocracy.com/policy.html#4) is 30 minutes/points
  * These users are banned and won't be assigned:
    * @0pdd[/z](https://www.0crat.com/u/0pdd): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/G6LPQQV2P?q=%7Bjob%3A%27gh%3Aamihaiemil%2Fdocker-java-api%2354%27%7D) (restricted area)

**Comment by 0crat** [comment]
The job [#54](https://github.com/amihaiemil/docker-java-api/issues/54) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/G6LPQQV2P/771871068). The budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9). If the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html).

**Comment by llorllale** [comment]
@amihaiemil this puzzle clearly has an impediment on #45

**Comment by amihaiemil** [comment]
@llorllale right, sorry, I should not have put it in scope yet

**Comment by amihaiemil** [comment]
@0crat out

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/amihaiemil/docker-java-api/issues/54#issuecomment-374297414))

@amihaiemil GitHub issue is still open, won't close

**Comment by llorllale** [comment]
@amihaiemil this one is still on my agenda

**Comment by amihaiemil** [comment]
@llorllale I know; tbh I don't really know how to unassign you. In the past, Yegor did it with "out"... now it doesn't seem to work. Could you maybe register the impediment and wait for its completion? I know it's not convenient, you have that slot blocked, but I promise I'll pay more attention in the future, shouldn't happen again :)

**Comment by llorllale** [comment]
@0crat waiting on #45

**Comment by 0crat** [comment]
> @0crat waiting on #45 ([here](https://github.com/amihaiemil/docker-java-api/issues/54#issuecomment-374935359))

@llorllale The impediment for [#54](https://github.com/amihaiemil/docker-java-api/issues/54) was registered successfully by @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0pdd** [comment]
The puzzle `47-dbe85f1e` has disappeared from the source code, that's why I closed this issue.

**Comment by 0crat** [comment]
Order was finished: +30 points just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
The job [#54](https://github.com/amihaiemil/docker-java-api/issues/54) is now out of scope
